include/wx/hashmap.h

```diff
@@ -38,6 +38,10 @@
     #define WX_HASH_MAP_NAMESPACE std::tr1
 #endif
 
+#if defined(_MSC_VER) && _MSC_VER > 1925
+    #define _WX_NO_STL_WITH_DECL
```

Comment:
Sorry, I don't understand this: didn't we decide that we should never mark these classes as DLL-exported, for all compilers, as they had only inline methods anyhow?

If I'm missing something and we still need to export them, I think this could be done in a simpler way by replacing
```
CLASSEXP CLASSNAME                                                           
```
line with some
```
WX_HASH_APPLY_EXPORT(CLASSEXP) CLASSNAME
```
and then defining `WX_HASH_APPLY_EXPORT` macro as just doing nothing for MSVC >= 1926.

But, again, it would be best not to export them at all, AFAICS...

Reply:
I tried to follow your suggestion in https://github.com/wxWidgets/wxWidgets/pull/1993#issuecomment-663274123
> I think we should leave _WITH_DECL but ignore DECL in them for MSVC (maybe just MSVC 19+). This should be an even more minimal change.

Maybe I understood wrong? I don't mind ignoring it for all. But I'll drop the commit from this PR and create a new one.

Reply:
See #2009

Reply:
Sorry, I forgot that I wrote "for MSVC". But looking at the code now it seems that it should be fine to do it for everybody. We do have other fully-inline classes that are not DLL-exported already and I don't remember having any problems with them...

Thanks again for working on this and sorry again for confusion/wrong suggestion!